api/IDBObjectStore.json

```diff
@@ -285,6 +285,39 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options_locale_parameter": {
+          "__compat": {
+            "description": "<code>options.locale</code> parameter",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "43"
```

Comment:
FYI, not gated by preference in FF, and never has been. Though much of the other locale stuff is.